_api-reference/nodes-apis/nodes-stats.md

```diff
@@ -1282,6 +1284,16 @@ total_rejections_breakup_shadow_mode.throughput_degradation_limits | Integer | T
 enabled | Boolean | Specifies whether the shard indexing pressure feature is turned on for the node.
 enforced | Boolean | If true, the shard indexing pressure runs in enforced mode (there are rejections). If false, the shard indexing pressure runs in shadow mode (there are no rejections, but statistics are recorded and can be retrieved in the `total_rejections_breakup_shadow_mode` object). Only applicable if shard indexing pressure is enabled. 
 
+### `cluster_manager_throttling`
+
+The `cluster_manager_throttling` object contains statistics about throttled tasks on the cluster manager node. It is populated only for actual cluster manager node.  
```

Comment:
```suggestion
The `cluster_manager_throttling` object contains statistics about throttled tasks on the cluster manager node. It is populated only for nodes that have a cluster manager role.  
```

Reply:
I think this has a bit different meaning. I will double-check but the original meaning was that the statistics will be provided only for the node that is currently elected as a cluster-manager. Not for all cluster-manager eligible nodes.

Reply:
Thank you for clarifying. Let me update the suggestion.